tests/test_input_validation.py

```
@@ -0,0 +1,698 @@
+            TelemetryData.validate({'voltage': 'invalid'})
+
+        # Test policy decision validation error
+        with pytest.raises(ValidationError):
+            PolicyDecision.validate({'invalid': 'data'})
+
+        # Test phase validation error
+        with pytest.raises(ValidationError):
+            MissionPhaseValidator.validate_phase('INVALID_PHASE')
+
+
+class TestTelemetryData:
```

Ruff (0.14.10)

Redefinition of unused `TestTelemetryData` from line 17

(F811)